tools/add_triage_signature.py

```diff
@@ -325,6 +333,78 @@ def _update_ticket(self, url, issue_key, should_update=False):
         self._update_description(issue_key, description)
 
 
+class FailureDetails(Signature):
```

Comment:
Not sure the signature responsibility is understandable with this name
It's mainly doing some kind of migration from labels to the new fields and from features to labels

Reply:
not only migration
It will handle the settings of the the various non-description fields for new tickets as well.
It does have a special case to handle migration

Reply:
So I think it's important to document those functions in a docstring

Reply:
added doc